docs/reference/inference/service-elasticsearch.asciidoc

```diff
@@ -51,6 +51,26 @@ include::inference-shared.asciidoc[tag=service-settings]
 These settings are specific to the `elasticsearch` service.
 --
 
+`adaptive_allocations`:::
+(Optional, object)
+include::{es-ref-dir}/ml/ml-shared.asciidoc[tag=adaptive-allocation]
```

Comment:
The text you include here contains:
```
(...) the number of allocations of the trained model is set (...)
```

I'm not sure if "trained model" should be mentioned here. It sounds like to predecessor of the inference service to me. I don't know, naming is hard and our names change at the time...

Reply:
I shortened it to `model` here. WDYT?